.github/workflows/AFTER_MERGE.yml

```diff
@@ -0,0 +1,24 @@
+name: AFTER_MERGE
+on:
+  push:
+    branches:
+    - "master"
+
+jobs:
+  Create_pull_request:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create pull request to develop
+      uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3
```

Comment:
Is it intended we use a commit hash for `repo-sync/pull-request`?

Reply:
Yup. This is to avoid a potential malicious action within the repository. GitHub recommends this if you can't trust the 3rd party: https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions

Reply:
I was able to review the code from the exact commit but can't tell whether the future commits are safe.